src/popup/router/components/Modal.vue

```diff
@@ -56,8 +56,8 @@
           class="footer"
         >
           <slot name="footer" />
-          <NodeConnectionStatus v-if="fullScreen" />
         </div>
+        <NodeConnectionStatus v-if="fullScreen" />
```

Comment:
Maybe we could have this component only in App scope positioned above the modal?

Reply:
I did not introduce it in that PR, and i don't think we have time for now to change the approach.